docs/addons/container-insights.md

```diff
@@ -43,12 +49,17 @@ daemonset.apps/cloudwatch-agent     1         1         1       1            1
 daemonset.apps/fluentd-cloudwatch   1         1         1       1            1           <none>          106s
 ```
 
-To enable or disable control plane logs with the console, please run the following command in your terminal - `aws eks update-cluster-config \
+To enable or disable control plane logs with the console, run the following command in your terminal.
+
+```bash
+aws eks update-cluster-config \
     --region us-east-2 \
     --name east-dev \
-    --logging '{"clusterLogging":[{"types":["api","audit","authenticator","controllerManager","scheduler"],"enabled":true}]}'`
+    --logging '{"clusterLogging":[{"types":["api","audit","authenticator","controllerManager","scheduler"],"enabled":true}]}'
```

Comment:
what happened to the PR from Aus team claiming they have this implemented with a custom resource?

Reply:
not sure. No PRs from them yet...